app/views/spree/shared/_review_summary.html.erb

```diff
@@ -1,9 +1,8 @@
 <div id="reviews">
-  <%= render 'spree/shared/rating', locals: @product, review: 0 %>
-  <% for review in (Spree::Reviews::Config[:track_locale] ? @product.reviews.localized(I18n.locale) : @product.reviews).default_approval_filter.preview %>
-    <%= render 'spree/shared/review', review: review %>
-  <% end %>
-  <% if Spree::Reviews::Config[:feedback_rating] && (!Spree::Reviews::Config[:require_login] || spree_current_user) %>
-    <%= link_to Spree.t(:write_your_own_review), new_product_review_path(@product), class: 'btn btn-default' %>
+  <% if Spree::Reviews::Config[:include_unapproved_reviews] == false and @product.reviews.approved.count == 0 %>
```

Comment:
Please remove the check "Spree::Reviews::Config[:include_unapproved_reviews] == false and @product.reviews.approved.count == 0" and just render this partial here.
<%= render 'spree/shared/rating', product: @product, review: 0 %>

Remove this partial:

<%= render 'spree/reviews/stars', stars: 0 %>

Reply:
when i remove the check it no show the empty stars when reviews are zero